dali/pipeline/data/types.cc

```diff
@@ -37,28 +37,6 @@ std::unordered_map<std::type_index, DALIDataType> TypeTable::type_map_;
 std::unordered_map<size_t, TypeInfo> TypeTable::type_info_map_;
 int TypeTable::index_ = DALI_DATATYPE_END;
 
-template <>
-void TypeInfo::Construct<CPUBackend>(void *ptr, Index n) {
```

Comment:
Worth mentioning that we found out that `Construct` and `Destruct` were actually not used anywhere.